sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/ResourceUtils.java

```diff
@@ -320,6 +320,7 @@ public static Resource getExpectedRanges(
                 .getLabelsBuilder()
                 .addLabels(getVIPLabel(vipName, vipPort));
 
+        builder.setVisibility(DiscoveryInfo.Visibility.CLUSTER);
```

Comment:
Maybe add a comment saying that this is meant to update visibility from prior service versions (it looks like that's what this is for?)